documentation/style-guide.rst

```diff
@@ -54,10 +54,15 @@ starting it with a lowercase letter should be avoided.
 Many special names are used in the Python documentation, including the names of
 operating systems, programming languages, standards bodies, and the like. Most
 of these entities are not assigned any special markup, but the preferred
-spellings are given here to aid authors in maintaining the consistency of
-presentation in the Python documentation.
+spellings are given in :ref:`glossary` to aid authors in maintaining the
+consistency of presentation in the Python documentation.
 
-Other terms and words deserve special mention as well; these conventions should
+.. _glossary:
+
+Specific words
+==============
```

Comment:
I predict this will confuse me, shall we have the same name for both the link and the title?

Reply:
There is also a [`glossary` directive](https://devguide.python.org/documentation/markup/#roles) (and a `:term:` role) that might add to the confusion.

Reply:
I changed the reference to `specific words`.